### PR TITLE
set default driver for the website to the lastest release

### PR DESCRIPTION
**Pull request by Issif**
Signed-off-by: Thomas Labarussias <issif+github@gadz.org>

This PR just updates the drivers website [https://download.falco.org/driver/site/index.html](https://download.falco.org/driver/site/index.html) to show the latest version of libs as default

### CODE REVIEW

**Comment by FedeDP** [comment]
/hold
Until #885 is merged otherwise pre-submit jobs will have to re-run before merging.

File: images/update-drivers-website/index.html
Comment (FedeDP) [inline images/update-drivers-website/index.html:47]: Isn't there a way to compute latest release given a semver? I mean, can we automate this?

Comment (Issif) [reply images/update-drivers-website/index.html:47]: Maybe by extracting the lib version used in last falco release, do you have the link where it's set for compilation?

**Review by maxgio92 — APPROVED** [review]
/approve

**Pull request merged**
